app/constants/urls.js

```diff
@@ -1,12 +1,12 @@
 import ENV from 'website-www/config/environment';
 
 export const APPS = {
-  HOME: ENV.APPS.HOME,
-  WELCOME: ENV.APPS.WELCOME,
-  EVENTS: ENV.APPS.EVENTS,
-  MEMBERS: ENV.APPS.MEMBERS,
-  STATUS: ENV.APPS.STATUS,
-  PROFILE: ENV.APPS.PROFILE,
+  HOME: ENV.APPS?.HOME,
+  WELCOME: ENV.APPS?.WELCOME,
+  EVENTS: ENV.APPS?.EVENTS,
+  MEMBERS: ENV.APPS?.MEMBERS,
+  STATUS: ENV.APPS?.STATUS,
+  PROFILE: ENV.APPS?.PROFILE,
```

Comment:
why not directly write 
```suggestion
const APPS = {...ENV.APPS, live}
  ```